### ISSUE TITLE
Prune build process

### ISSUE THREAD

**Issue by HendrikStrobelt** [ieee-vgtc/ieeevis.org#1406]
The current building pipeline requires knowledge about Jekyll (Ruby, Gems, Docker,... ) and Webpack (NPM, SASS, NodeJS,...). This makes onboarding of new chairs very very hard and also requires a stack of dependencies that is hard to maintain . Creating a set of static webpages should not be that complex. Maybe focus on a webpack or python stack only.

**Comment by HendrikStrobelt** [comment]
Another example:
- Where and how is `governance/` served ?
- Why is there one VUE component ?
- ..

**Comment by spetruzza** [comment]
Agreed, for example if we could go down to simple github pages, the entire build and deployment process would become trivial... We'll talk about it.

**Comment by yelper** [comment]
Vue component was made by the designer we hired to redesign; it handles the navbar and the accessibility for it.

Governance is placed by the last-built branch, then "placed" by calling https://github.com/ieee-vgtc/ieeevis.org/blob/vis2022/scripts/fix_file_extensions.sh via make2022 (etc)
